store/cachekv/store.go

```diff
@@ -378,11 +378,14 @@ func (store *Store) clearUnsortedCacheSubset(unsorted []*kv.Pair, sortState sort
 		if item.Value == nil {
 			// deleted element, tracked by store.deleted
 			// setting arbitrary value
-			store.sortedCache.Set(item.Key, []byte{})
+			if err := store.sortedCache.Set(item.Key, []byte{}); err != nil {
+				panic(err)
```

Comment:
is this the line that fixes the bug?

Reply:
No. The fix is breaking up the nested for loops into 2 separate single for loops

Reply:
This is just a patch while I was debugging.

Reply:
is this a consensus-breaking change?